src/Conversions.cc

```diff
@@ -1326,3 +1326,130 @@ math::AxisAlignedBox ignition::gazebo::convert(const msgs::AxisAlignedBox &_in)
   out.Max() = msgs::Convert(_in.max_corner());
   return out;
 }
+
+//////////////////////////////////////////////////
+template<>
+IGNITION_GAZEBO_VISIBLE
+msgs::ParticleEmitter ignition::gazebo::convert(const sdf::ParticleEmitter &_in)
+{
+  msgs::ParticleEmitter out;
+  out.set_name(_in.Name());
+  switch(_in.Type())
+  {
+    default:
+    case sdf::ParticleEmitterType::POINT:
+      out.set_type(msgs::ParticleEmitter::POINT);
+      break;
+    case sdf::ParticleEmitterType::BOX:
+      out.set_type(msgs::ParticleEmitter::BOX);
+      break;
+    case sdf::ParticleEmitterType::CYLINDER:
+      out.set_type(msgs::ParticleEmitter::CYLINDER);
+      break;
+    case sdf::ParticleEmitterType::ELLIPSOID:
+      out.set_type(msgs::ParticleEmitter::ELLIPSOID);
+      break;
+  }
+
+  msgs::Set(out.mutable_pose(), _in.RawPose());
+  msgs::Set(out.mutable_size(), _in.Size());
+  msgs::Set(out.mutable_particle_size(), _in.ParticleSize());
+  out.mutable_rate()->set_data(_in.Rate());
+  out.mutable_duration()->set_data(_in.Duration());
+  out.mutable_emitting()->set_data(_in.Emitting());
+  out.mutable_lifetime()->set_data(_in.Lifetime());
+  if (_in.Material())
+  {
+    out.mutable_material()->CopyFrom(convert<msgs::Material>(*_in.Material()));
+  }
+  out.mutable_min_velocity()->set_data(_in.MinVelocity());
+  out.mutable_max_velocity()->set_data(_in.MaxVelocity());
+  msgs::Set(out.mutable_color_start(), _in.ColorStart());
+  msgs::Set(out.mutable_color_end(), _in.ColorEnd());
+  out.mutable_scale_rate()->set_data(_in.ScaleRate());
+  out.mutable_color_range_image()->set_data(_in.ColorRangeImage());
+
+  if (!_in.ColorRangeImage().empty())
```

Comment:
I tested without the color range image set in sdf and this returns `__default__` and causes system paths to print an error msg about not being able to find the file. I think we can set the default value in sdf to empty string?

Reply:
Changed SDF to use an empty string in https://github.com/osrf/sdformat/pull/528/commits/5d9fc53ae51bb6c45ebfb59b664f85f21d65ef18